### ISSUE TITLE
Angular 10 warning

### ISSUE THREAD

**Issue by spock123** [btroncone/ngrx-store-localstorage#160]
When compiling with Angular10, the following warning is displayed:

```
WARNING in /Users/lrj/project/club-osb/src/app/store/meta-reducers/index.ts depends on ngrx-store-localstorage. 

CommonJS or AMD dependencies can cause optimization bailouts.

For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

```

Is it possible to do es6 build? Thank you

Angular: 10.0.1
Ngrx-store-localstore:  ^9.0.0

**Comment by conor-mac-aoidh** [comment]
Why not build with `ng-packagr` ?

**Comment by BBlackwo** [comment]
@conor-mac-aoidh that's a great idea to use `ng-packgr`.

If someone can submit a PR to update the build system I'd be very happy to review it.

**Comment by darkedges** [comment]
@BBlackwo I have created https://github.com/btroncone/ngrx-store-localstorage/pull/162 to help resolve this.

been testing using https://www.npmjs.com/package/@darkedges/ngrx-store-localstorage/v/9.0.2

**Comment by geo242** [comment]
@darkedges Thanks for releasing that! So far it's working well in my testing and the warning is gone.

@BBlackwo Will this be merged in soon?

**Comment by trinitiwowka** [comment]
Hi @BBlackwo 
Will this be merged in soon? 9.0.2

**Comment by BBlackwo** [comment]
Sorry it took me a while to get this fixed everyone. Should be fixed now thanks to @darkedges 🙂
